.changeset/fluffy-actors-listen.md

```diff
@@ -0,0 +1,32 @@
+---
+'@leafygreen-ui/text-input': major
+---
+
+[LG-4145](https://jira.mongodb.org/browse/LG-4145)
+
+`aria-label` and `aria-labelledby` props are passed to the input.
+- a custom `aria-labelledby` attribute will only be set on the input if `label` prop is undefined. Otherwise, the generated label component id will be used for the `aria-labelledby` attribute
```

Comment:
I'm a little confused by this -- If there is no `label` then we're adding the `aria-labelledby` attribute?

Reply:
probably better to use the word "can" than "will"

I can add an example to the changelog

Reply:
updated and added examples